scripts/rtma3d/exrtma3d_obsprep_radar.ksh

```diff
@@ -123,6 +123,7 @@ if [ -s filelist_mrms ]; then
    mv filelist_mrms filelist_mrms_org
    ls MergedReflectivityQC_*_${YYYY}${MM}${DD}-${HH}????.grib2 > filelist_mrms
    numgrib2=`more filelist_mrms | wc -l`
+   cpreq ${COMINradar}/${obsname}/${obsname}_*_${YYYY}${MM}${DD}-${HH}${min}*.grib2.gz ${COMINobsproc_rtma3d}
```

Comment:
These are copying the wrong files for some reason. If you look in your obsprd.* directories, they are copying the 2 minute files instead instead of the 0 minute files (which are the files that are used).